fastlane/Fastfile

```diff
@@ -54,16 +54,39 @@ platform :ios do
   end
   
   desc "Upload a local IPA to Crashlytics with (group)."
+  desc "#### Options"
+  desc " * **`generate_changelog`**: Whether or not to generate a changelog as the release notes (boolean – default: `false`)"
+  desc " * **`changelog_type`**: The type of changelog to generate (e.g. `\"git\"`, `\"jenkins\"`, `\"pr\"` – default: `\"git\"`)"
+  desc " * **`release_notes`**: A string to set as the release notes; overrides any generated changelog."
   lane :uploadToCrashlytics do |options|
+    options[:generate_changelog] ||= false
+    options[:changelog_type] ||= "git"
+
     raise "uploadToCrashlytics: A Crashlytics group must be passed as an option." unless options[:group]
-    
-    if options[:generate_changelog] == true 
-      changelog_from_git_commits
+
+    unless options.key?(:release_notes)
+      if options[:generate_changelog] == true
+        case options[:changelog_type]
+          when "jenkins"
+            make_changelog_from_jenkins
+
+          when "pr"
+            UI.error("uploadToCrashlytics: Environment variable for PR ID is not set (ghprbPullId).") unless ENV.key?("ghprbPullId")
```

Comment:
Wasn't the `ENV.key?` syntax added in a fairly recent version of Ruby? I'm fine with adding it, but there may need to be something which checks the version of Ruby being called. I could be wrong though.

Reply:
It looks like this has been available since ruby 2.1.4: https://ruby-doc.org/core-2.1.4/ENV.html#method-c-key-3F